### PR TITLE
Dockerize ridgeback

### PR DESCRIPTION
**Pull request by andresilva**
This PR adds support for packaging `ridgeback` as a Docker container. The grunt build process was integrated into an `sbt` task which is automatically executed when packaging the app.

Additionally, the `src/main/webapp/build` was added as an unmanaged resource directory. This way, when packaging the backend, the dashboard assets will be packaged inside the JAR.

I also changed the default database location, so that its easier to create it in an external volume when running inside a container.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,27 +7,37 @@ Ridgeback is a continuous integration service for performance tests.
...
+
+This will create the database in a local folder `db` so that the database state can be preserved.
+
 ## Introduction
 
 While performance may not be the most important factor and benchmarks always need to be taken with a grain of salt, performance is too big of a deal to be simply ignored in any medium to large scale project. Ridgeback aims at tracking performance in the evolution of software projects, using a continuous integration approach.
 
 Tests can be triggered by writing a comment on a pull request with a certain keyword (usually PERFTESTS) and Ridgeback proceeds to clone the repository, fetch the pull request commits and run what is defined in the `.perftests.yml` file on the root directory of the repository. Once this step is done, the duration or durations of the jobs are stored in the database (currently a SQLite file) for posterior analysis. Quickly afterwards, the bot replies to the pull request with a quick overview of the tests ran and a comparison to previous tests (see `messageTemplate` below).
 
-An overview of all projects, history of tests and jobs can be seen in the provided dashboard (by default, http://localhost:8080/gui) (screenshots below!).
```
Comment (ruippeixotog) [inline README.md:30]: http://localhost:8080/gui works too, doesn't it?

Comment (andresilva) [inline README.md:30]: Only `http://localhost:8080/gui/`. But I'll fix that ;-)

**Comment by ruippeixotog** [comment]
If only you had fixed that trailing slash... 😄

**Comment by jcazevedo** [comment]
> If only you had fixed that trailing slash... 😄

lol'd

**Comment by andresilva** [comment]
Cleaning up my open PR list 😛 Unfortunately this project never became the smash hit we expected.

**Pull request closed without merging**